operator/pkg/controllers/master.go

```diff
@@ -75,3 +83,61 @@ func (m *MasterManager) createPod(r *ElasticJobReconciler, job *elasticv1alpha1.
 	}
 	return nil
 }
+
+func (m *MasterManager) createPodName(job *elasticv1alpha1.ElasticJob) string {
```

Comment:
I think `generatePodName` should be better.

Reply:
Done